tests/encryption_/models.py

```diff
@@ -0,0 +1,65 @@
+from django_mongodb_backend.encryption import EqualityQuery, RangeQuery
+from django_mongodb_backend.fields import (
+    EncryptedBigIntegerField,
+    EncryptedBinaryField,
+    EncryptedBooleanField,
+    EncryptedCharField,
+    EncryptedDateField,
+    EncryptedDateTimeField,
+    EncryptedDecimalField,
+    EncryptedEmailField,
+    EncryptedFloatField,
+    EncryptedGenericIPAddressField,
+    EncryptedIntegerField,
+    EncryptedTextField,
+    EncryptedTimeField,
+    EncryptedURLField,
+)
+from django_mongodb_backend.models import EncryptedModel
+
+
+class Appointment(EncryptedModel):
+    time = EncryptedTimeField(queries=EqualityQuery())
+
+
+class Billing(EncryptedModel):
+    cc_type = EncryptedCharField(max_length=20, queries=EqualityQuery())
+    cc_number = EncryptedBigIntegerField(queries=EqualityQuery())
+    account_balance = EncryptedDecimalField(max_digits=10, decimal_places=2, queries=RangeQuery())
+
+    class Meta:
+        db_table = "billing"
```

Comment:
I left a comment about this before, but we shouldn't use an explicit `db_table` (unless it's prefixed with the app name) since that creates that possibility that the table names in two different test apps could collide. Is a custom name really needed?